SvgImage.js

```diff
@@ -1,46 +1,82 @@
 // @flow
 
+import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import { View, WebView } from 'react-native';
+import { View, WebView, StyleSheet } from 'react-native';
+const fetchingDebug = require("debug")("react-native-remote-svg:SvgImage:fetch");
+const fetchResultDebug = require("debug")("react-native-remote-svg:SvgImage:fetch:result");
```

Comment:
Hey, it looks like you used the `debug` package to debug this issue. Does it make sense to commit it onto master?